financeager/cli.py

```diff
@@ -35,16 +35,15 @@ def main():
     sys.exit(run(**_parse_command()))
 
 
-def run(command=None, config=None, verbose=False, **params):
-    """High-level API entry point.
-    All 'params' are passed to 'Client.safely_run()'.
-    'config' specifies the path to a custom config file (optional). If 'verbose'
-    is set, debug level log messages are printed to the terminal.
-
-    This function can be used for scripting. Provide 'command' and 'params'
-    according to what the command line interface accepts (consult help via
-    `financeager [command] --help`), e.g. {"command": "add", "name":
-    "champagne", "value": "99"}.
+def run(command=None, config_filepath=None, verbose=False, **params):
+    """High-level API entry point, useful for scripts.
+    Run 'command' passing 'params' according to
+    what the command line interface accepts
+    (consult help via `financeager [command] --help`),
+    e.g. {"command": "add", "name": "champagne", "value": "99"}.
+    All kwargs are passed to 'communication.run()'.
+    'config_filepath' specifies the path to a custom config file (optional).
+    If 'verbose' is set, debug level log messages are printed to the terminal.
```

Comment:
Sorry for noticing so late. The docstring should be identical to the one from the master. Can you update it once more?
You can also grant me permission to contribute to the pull-request from inside your PR settings if you want :)